index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>📘 The Code Magazine</h1>
+    <h2><b>The Basic Language of the Web: HTML</b></h2>
+    <img src="img/laura-jones.jpg" alt="portretul lui laura jones" />
```

Comment:
daca ai setat la html limba engleza, ar fi bine si la alt comentariile sa fie in engleza